src/backend/utils/indexer.js

```diff
@@ -56,7 +62,7 @@ const search = async (textToSearch) => {
       simple_query_string: {
         query: textToSearch,
         default_operator: 'and',
-        fields: ['text'],
+        fields: ['text', 'title', 'author'],
```

Comment:
I wonder if we should choose between fields to address different filters.
```suggestion
        fields: filter === 'post' ? ['text'] : ['author'],
```

Reply:
I also wonder that we need the feature? Then we can do it.
I looked at the code and I realized that for author, the search used graphql~.
Please let me clear~
I thought the reason the fields are added in ES is checking more fields only for the post case.

Reply:
Yes, for `author` we use graphql, but we're in the process of removing it and we will probably use ES instead.
Adding `author` here is fine, it's just that I think we should add it like in my suggestion so we can search by `text` **or** `author` depending on the selected filter in the front-end. The way you add it means that ES will try to find the text in the posts **and** and the authors, which is something that might be added in the future, but currently doesn't match the way the front-end works.